registry/registryprovider.go

```diff
@@ -143,21 +163,21 @@ type githubRegistryProvider struct {
 // NewGithubRegistryProvider creates a GithubRegistryProvider.
 func NewGithubRegistryProvider(cp common.CredentialProvider) GithubRegistryProvider {
 	if cp == nil {
-		panic(fmt.Errorf("CP IS NIL: %v", cp))
+		panic(fmt.Errorf("cp is nil: %v", cp))
```

Comment:
The poor developer who sees this message may not know what `cp` means without some digging. It would make life easier for them to make the text a bit more explanatory, such as `no credential provider`. Also, if cp is nil, why print it?